mandatory/4-tax.js

```diff
@@ -17,8 +25,12 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
-
+function addTaxAndFormatCurrency(price) {
+  var priceInclusiveOfTax = calculateSalesTax (price);
```

Comment:
Tiny typo here - there shouldn't be a space when you call a function (however it will work!)